diff --git a/node_modules/@clr/angular/fesm2022/clr-angular-forms-number-input.mjs b/node_modules/@clr/angular/fesm2022/clr-angular-forms-number-input.mjs
index dd7bc0e..ccf86b2 100644
--- a/node_modules/@clr/angular/fesm2022/clr-angular-forms-number-input.mjs
+++ b/node_modules/@clr/angular/fesm2022/clr-angular-forms-number-input.mjs
@@ -115,7 +115,7 @@ i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "21.1.3", ngImpor
                     host: {
                         '[class.clr-form-control]': 'true',
                         '[class.clr-form-control-disabled]': 'control?.disabled',
-                        '[class.clr-form-control-readonly]': 'input.readonly',
+                        '[class.clr-form-control-readonly]': 'input?.readonly',
                         '[class.clr-row]': 'addGrid()',
                     },
                     providers: [FormsFocusService, NgControlService, ControlIdService, ControlClassService],
